ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-custom-control-plane-policy-edge-1.cfg

```diff
@@ -14,8 +14,6 @@ flow tracking hardware
 !
 service routing protocols model multi-agent
 !
-ip as-path access-list ASPATH-WAN permit 65000 any
```

Comment:
This change is a breaking change right? Maybe a missed revert or need another if/else for HA case.

Reply:
Hmm missing some if yes

Reply:
well actually no it is a fix. This was not being used anywhere before